pkg/storage/buckets/store.go

```diff
@@ -166,27 +182,82 @@ func (b *BucketStore) List(ctx context.Context, startAfter, endBefore string, li
 }
 
 // convertMeta converts stream metadata to object metadata
-func convertMeta(m objects.Meta) (Meta, error) {
-	var cipher storj.Cipher
-
-	pathEncType := m.UserDefined["path-enc-type"]
+func convertMeta(m objects.Meta) (out Meta, err error) {
+	out.Created = m.Modified
+	// backwards compatibility for old buckets
+	out.PathEncryptionType = storj.AESGCM
+	out.EncryptionScheme.Cipher = storj.Invalid
 
-	if pathEncType == "" {
-		// backward compatibility for old buckets
-		cipher = storj.AESGCM
-	} else {
+	if pathEncType := m.UserDefined["path-enc-type"]; pathEncType != "" {
```

Comment:
Shouldn't we get values from the map using the pattern `value, ok := map["key"]` to avoid panics if the key does not exist?

Reply:
Accessing a key that doesn't exist won't cause a panic, it will only return a zero-value. The behavior we want if an empty string is really stored is the same behavior we want if the key does not exist, so I think this is probably best still